pkg/crc/machine/machine.go

```diff
@@ -321,11 +329,6 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 	// Additional steps to perform after newly created VM is up
 	if !exists {
-		logging.Info("Generating new SSH key")
-		if err := updateSSHKeyPair(sshRunner); err != nil {
-
-			return startError(startConfig.Name, "Error updating public key", err)
-		}
 		// Copy Kubeconfig file from bundle extract path to machine directory.
 		// In our case it would be ~/.crc/machines/crc/
```

Comment:
This commit avoids issues with the ssh key not being available, but won't we be getting issues with a missing kubeconfig where we were getting issues with the missing ssh key before?

Reply:
Yes, we'd be getting issues with that, makes sense to copy the kubeconfig also earlier..